handbook/handbook.concepts.md

```diff
@@ -2,16 +2,20 @@
 id: BMRfppP5dBMBl9v7IOyf4
 title: Concepts
 desc: ''
-updated: 1635831886673
+updated: 1643646745124
 created: 1630624424196
 ---
 
-## Lookup
+### Dendron Community
+
+The Dendron community consists of both the humans that use Dendron and the team members working on it.
+
+### Lookup
 
 The process of building on knowledge that already exists.
 
 
-## Recovery Phase
+### Recovery Phase
```

Comment:
Should the headers in this file each be H2 `##` instead of H3 `###`? Or is `###` being used purposely for something like note ref embedding?

Reply:
yes, good catch. will update in a separate commit